packages/votingV2/src/mappings/votingV2.ts

```diff
@@ -500,6 +500,10 @@ export function handleUpdatedReward(event: UpdatedReward): void {
   user.nextIndexToProcess = nextIndexToProcessChain;
 
   if (nextIndexToProcessChain.gt(globals.maxNextIndexToProcess)) {
+    // This value can be compared to the users' nextIndexToProcess to see if the users'
+    // trackers are up to date. This is also demonstrated by the user.cumulativeSlash versus
+    // user.cumulativeCalculatedSlash comparison; if they differ, the user's trackers are out of date.
+    // It should be noted that user.cumulativeCalculatedSlash is always updated for all users.
```

Comment:
Note that I've added the `global.maxNextIndexToProcess` to see at a glance if a user has updated his trackers recently.

`user.cumulativeCalculatedSlash` vs `user.cumulativeSlash` comparison can also be used to know this.

`user.cumulativeCalculatedSlash` should be the one used in the UI as it's always up to date thanks to [updateUsersSlashingTrackers](https://github.com/UMAprotocol/subgraphs/blob/3c510acc5025c72b39fb20e7b5b0da2b41820fcf/packages/votingV2/src/mappings/votingV2.ts#L153)